forte/data/base_store.py

```diff
@@ -31,6 +32,81 @@ def __init__(self):
         various fields stored in entry lists.
         """
 
+    def __getstate__(self):
+        state = self.__dict__.copy()
+        return state
+
+    def __setstate__(self, state):
+        self.__dict__.update(state)
```

Comment:
These `getstate` `setstate` looks like the default implementation? Then probably no need to have them here

Reply:
Yes sure

Reply:
I just noticed I might need to keep `getstate` because I need to call it in `to_string`